.github/workflows/docs.yml

```diff
@@ -1,4 +1,4 @@
-name: Docs
+name: Documentations
```

Comment:
Should be "Documentation", idk why it's abbreviated "Docs" but it doesn't actually have a plural form in English ^^

Reply:
Ah!
Fixed to “Documentation”, and I've renamed the file `documentation.yml`.